lib/src/track/local/video.dart

```diff
@@ -49,9 +53,13 @@ class LocalVideoTrack extends LocalTrack with VideoTrack {
   /// Note: Android requires a foreground service to be started prior to
   /// creating a screen track. Refer to the example app for an implementation.
   static Future<LocalVideoTrack> createScreenShareTrack([
-    ScreenShareTrackOptions? options,
+    VideoCaptureOptions? options,
   ]) async {
-    options ??= const ScreenShareTrackOptions();
+    if (options is VideoCaptureOptions &&
+        options is! ScreenShareCaptureOptions) {
+      options = ScreenShareCaptureOptions.from(captureOptions: options);
+    }
+    options ??= const ScreenShareCaptureOptions();
```

Comment:
Oh looks like this is overriding default options if it is not screen share options. Seems a bit confusing. Is it possible to just set the defaults to the right values in the caller?

Reply:
I'm trying to keep it consistent with this pattern 👉 https://github.com/livekit/client-sdk-flutter/blob/main/lib/src/participant/local_participant.dart#L88-L100

For now .. but I think we should move the create track methods to
`LocalParticipant.createCameraTrack` so there is access to the default room options.